OpenRA.Mods.Common/Projectiles/Bullet.cs

```diff
@@ -235,6 +235,9 @@ bool ShouldExplode(World world)
 
 			if (flightLengthReached && shouldBounce)
 			{
+				if (!world.Map.Contains(world.Map.CellContaining(pos)))
```

Comment:
Can we move the `terrainPos` variable above this check and reuse it instead of querying `CellContaining` twice?

Reply:
Ah, yes I got confused as it was oddly named.